src/main/java/net/playeranalytics/extension/CMI/CMIEcoEventListener.java

```diff
@@ -0,0 +1,30 @@
+package net.playeranalytics.extension.CMI;
+
+import com.Zrips.CMI.events.CMIUserBalanceChangeEvent;
+import com.djrapitops.plan.extension.Caller;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.plugin.Plugin;
+
+public class CMIEcoEventListener implements Listener {
+    private final Caller caller;
+
+    public CMIEcoEventListener(Caller caller) {
+        this.caller = caller;
+    }
+
+    public static void register(Caller caller) {
+        Plugin plan = Bukkit.getPluginManager().getPlugin("Plan");
```

Comment:
This can use `ListenerService.getInstance().registerListenerForPlan(this);`

Reply:
Reworked with af3cba6112d795aa66fd84568b6ec7d0447b2df8